perllib/FixMyStreet/Script/Alerts.pm

```diff
@@ -103,6 +106,14 @@ sub send() {
             my $url = $cobrand->base_url_for_report($row);
             # this is currently only for new_updates
             if (defined($row->{item_text})) {
+                # this might throw up the odd false positive but only in cases where the
```

Comment:
Won't it also output on the first update of every report? The logic should probably match how the templates do it to try and cover this.

Reply:
It will yes, but if it's a blank update then that means the state has changed so we should output it and if not then we'd output it anyway. The false positive is it might mention the status change when there hasn't been one which I'm not really sure how to avoid.

Reply:
Sorry I wasn't clear, I meant that yes, for every normal first update with text and no state change, it will also always say "Problem marked as confirmed" which doesn't feel right. The template has the additional conditional of ignoring a problem_state of confirmed while last_problem_state is the empty string, which would also work here unless I'm missing something?